### PR TITLE
Walks x times generating paths depending on the heuristic

### LINKED ISSUE #57: System for finding Entanglement data from Heuristics

### PR DESCRIPTION
**Pull request by jamadaha**
- closes #57

### CODE REVIEW

File: src/Reformulators/RandomWalkerReformulator.cpp [hunk 1]
```diff
@@ -2,15 +2,23 @@
 
 using namespace std;
 
-PDDLInstance RandomWalkerReformulator::ReformulatePDDL(PDDLInstance* instance) {
+// Move this to config?
+// Or generate it based on something?
+#define RUN_COUNT 10
```
Comment (kris701) [inline src/Reformulators/RandomWalkerReformulator.cpp:7]: Might be a good idea to merge #103 before this then

Comment (jamadaha) [reply src/Reformulators/RandomWalkerReformulator.cpp:7]: Oh, yeah. Right.

Comment (kris701) [reply src/Reformulators/RandomWalkerReformulator.cpp:7]: You can just use the `ConstantWidth` one and set it to 10 for now until we get some better width functions

Comment (jamadaha) [reply src/Reformulators/RandomWalkerReformulator.cpp]: It is done

**Pull request merged**